src/arguments.js

```diff
@@ -54,7 +54,24 @@ const getDefaultValue = (description, regexp) => {
  */
 const getEnum = (string, context) => {
     const result = getValueByRegexp(string, context.regexp.enumValues.enum);
-    return result ? result[1].split(context.regexp.enumValues.split) : null;
+    return result ? removeExtraEnumValues(
+        result[2].split(context.regexp.enumValues.split), context) : null;
+};
+
+/**
+ * Remove extra enum values (such as 'file', 'path', 'folder', etc.)
+ * @param {array} enumArray - array with enum values
+ * @param {object} context - internal config
+ * @return {array} enumArray - enum without extra values or null if it empty
+ */
+const removeExtraEnumValues = (enumArray, context) => {
+    let result = [];
+    enumArray.forEach((value) => {
```

Comment:
I propose to change `forEach` to `filter`

Reply:
@svoboda-rabstvo done.